manim/mobject/mobject.py

```diff
@@ -997,35 +1117,39 @@ def scale(self, scale_factor: float, **kwargs) -> "Mobject":
         :meth:`move_to`
 
         """
-        if config["use_opengl_renderer"]:
+        if self.opengl:
             self.apply_points_function(
                 lambda points: scale_factor * points,
                 works_on_bounding_box=True,
                 **kwargs,
             )
-            return self
         else:
             self.apply_points_function_about_point(
                 lambda points: scale_factor * points, **kwargs
             )
-            return self
-
-    def rotate_about_origin(self, angle, axis=OUT, axes=[]):
+        return self
+        # self.apply_function_to_points(
```

Comment:
Again - just some commented code to be removed

Reply:
Thank you for your feedback. OpenGLMobject has been removed along with all the comments in the Mobject class.

Reply:
Removed.